back/src/pods/storage/api/storage.mock.ts

```diff
@@ -1,27 +1,16 @@
+import { ConnectSessionInfo, UserSession, VotesFromRooms } from 'dals/user';
 // This is just a demo approach, storing in memory session Info
```

Comment:
I would update this comment to 

```ts
// Only to be used on a local development environment (in case you don't want run the mongo db version of the API, e.g. no docker installed locally)
```